docs/explanations/reviews.md

```diff
@@ -0,0 +1,53 @@
+# Approved Reviewers
+
+Code reviews are an important tool for ensuring both quality and collective responsibility for the codebase. Dodal PRs *must* be reviewed by a member of [the approved review team](https://github.com/orgs/DiamondLightSource/teams/bluesky-reviewers) before they are merged. 
+
+## Review Standards and Practices
+
+When reviewing you should ensure the code meets the [repository standards](../reference/standards.rst) and [device standards](../reference/device-standards.rst). Approved reviewers will adhere to the following, if they do not, junior reviewers and reviewees should feel empowered to hold them to them.
+
+### Questions and Comments
+
+Questions and requests for clarification are encouraged. If you have comments for the reviewer you could add them with the following prefixes to be helpful:
+
+- Nit - This is arguably just taste, you don't mind if the developer doesn't follow this
```

Comment:
Maybe this is obvious enough without having in writing, but code should not be rejected just from `nit` comments